package.json

```diff
@@ -21,8 +21,8 @@
 		"release": "changeset publish",
 		"inspector": "npx @modelcontextprotocol/inspector node mcp-server/server.js",
 		"mcp-server": "node mcp-server/server.js",
-		"format-check": "prettier --check .",
-		"format": "prettier --write ."
+		"format-check": "node node_modules/.bin/biome format .",
+		"format": "node node_modules/.bin/biome format . --write"
```

Comment:
can't we just do biome format ? instead of doing node node modules etc... patch

Reply:
Yes, good catch. I think I was trying to follow some existing convention or something, IDK. I've pushed a commit to shorten this